activities/activity_johnpolo.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+"""
+Instructions
+- Functions intended to run for each scan
+  need to be named run_xxxxx
+
+- Do not modify the parameters of the run_xxx function
+  unless you know what you are doing
+  see optional parameters:
+  https://github.com/tangible-landscape/grass-tangible-landscape/wiki/Running-analyses-and-developing-workflows#python-workflows
+
+- All gs.run_command/read_command/write_command/parse_command
+  need to be passed env parameter (..., env=env)
```

Comment:
You can delete these instructions and replace them which once sentence description of what you are doing in the script.

See the second point before you delete the text - it relates to my other comment.

Reply:
I deleted the three statements and wrote a one sentence description. 

In response to the missing execution on the anna_petrasova.py file, I did the following in Git Bash:
john@DESKTOP-7OSMQHM MINGW64 /d/projects_and_data/data/landscape_git/gis714-2021-tangible-landscape (concave_activity_jp)
$ git update-index --chmod=+x activities/activity_anna_petrasova.py

It returned no errors, so I think it worked.